eldritch/monsters.py

```diff
@@ -131,7 +131,15 @@ def movement(self, state):
         return movement
     return "normal"
 
-  def get_interrupt(self, event, state):  # pylint: disable=unused-argument
+  def get_interrupt(self, event, state):
+    if (isinstance(event, events.TakeTrophy)
```

Comment:
I am debating the merits of using an interrupt vs hard-coding this into TakeTrophy. If you hard-code this into TakeTrophy, you have to make a distinction between the behavior of TakeTrophy and ForceTakeTrophy. If you use an interrupt, you have to make sure that all monsters that could possibly be endless call return super().get_interrupt() when they don't have any interrupts. I'm not sure that I prefer one over the other, so I'm just making a mental note that I may want to look at this again in the future.